src/pybind/mgr/dashboard/controllers/crush_rule.py

```diff
@@ -47,6 +47,16 @@ def create(self, name, root, failure_domain, device_class=None):
         }
         CephService.send_command('mon', 'osd crush rule create-replicated', **rule)
 
+    @RESTController.Resource('POST')
+    def create_erasure(self, name, profile, failure_domain, device_class):
+        rule = {
+            'name': name,
+            'profile': profile,
+            'type': failure_domain,
+            'class': device_class
+        }
+        CephService.send_command('mon', 'osd crush rule create-erasure', **rule)
+
```

Comment:
Instead of creating a separate API for erasure, can we combine it along with the existing one. It looks like both of them are almost similar with one exception. So we can toggle them based on `replication` or `erasure` type. That's the standard we have across the dashboard and can keep the APIs minimized.